### PR TITLE
Postgresql-backed geocode caching

### PR DESCRIPTION
**Pull request by rjmac**

### CODE REVIEW

File: src/main/resources/reference.conf [hunk 1]
```diff
@@ -16,21 +16,9 @@ com.socrata.geocoding-secondary {
...
-      port = 9160
-      max-connections-per-host = 1
-      connect-timeout = "500ms"
-    }
-  }
-
   geocoder {
     filter-multiplier = 1
     cache {
-      column-family = geocode_cache
```
Comment (catstavi) [inline src/main/resources/reference.conf:33]: what was this even used for?

Comment (rjmac) [reply src/main/resources/reference.conf:33]: The table name in cassandra used to be configurable.  That was never really used for anything, and with the switch to doing schema-management via liquibase it became impractical, so I deleted the ability to configure it (in either store).

**Comment by catstavi** [comment]
Ok, I can see where sample.conf is included, in application.conf and I get the docker conf file will be used by the dockerized version, but where does reference.conf come in?

**Comment by rjmac** [comment]
reference.conf is unconditionally used as a base config no matter where it's running.

**Comment by rjmac** [comment]
Yes, but part of the point of doing it this way is that no one needs to change immediately.

**Comment by rjmac** [comment]
(at least, if I've done it right!)

**Comment by catstavi** [comment]
(that last comment was also an overall approval, in case you missed it)

**Comment by rjmac** [comment]
I did, thanks for pointing it out. :)

**Pull request merged**